frontend/src/lib/utils.tsx

```diff
@@ -118,11 +118,12 @@ export function DeleteWithUndo(
     props: PropsWithChildren<{
         endpoint: string
         object: {
-            name: string
+            name?: string
             id: number
         }
         className: string
         style: CSSProperties
+        callback: CallableFunction
```

Comment:
CallableFunction is something weird/incorrect. `() => void` is often the right type.